SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -2266,6 +2268,26 @@ void ActorSystem::UpdateInterestComponent(AActor* Actor)
 		NetDriver->InterestFactory->CreateInterestUpdate(Actor, NetDriver->ClassInfoManager->GetOrCreateClassInfoByObject(Actor), EntityId);
 
 	NetDriver->Connection->SendComponentUpdate(EntityId, &Update);
```

Comment:
still sending old one here?

Reply:
should this be in an else case below?

Reply:
Separated this functionality (`interestDirty` and `clientInterestDirty` ) and made `UpdateClientInterest`